src/components/leaderboard/LeaderboardRow/index.tsx

```diff
@@ -1,4 +1,5 @@
 import Image from 'next/image';
+import { RefObject } from 'react';
```

Comment:
```suggestion
import type { RefObject } from 'react';
```
to leave TypeScript code out of compiled bundle